### DISCUSSION TITLE
Calling team_id from CLI

### DISCUSSION THREAD

**Discussion by szotp** [fastlane/fastlane#17109] [Q&A]
http://docs.fastlane.tools/actions/team_id/#team_id

How do I call this command from CLI? It does not document any parameters, I tried calling this:
```
fastlane run team_id 'Q2CBPK58CA'
```

but it still fails with
```
[!] Please pass your Team ID (e.g. team_id 'Q2CBPK58CA')
```

like it was not capable or parsing arguments? I worked around this by manually exporting all variables from https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/team_id.rb, but it's lame

**Comment by tmxkevingriesbach** [comment]
What are you trying to achieve? Do you need the team_id for match or something else?  
You can specify it in match or others simply by
```ruby
match(
  username: "SOMETHING",
  [...]
  platform: "ios",
  team_id: "ABCDEFG"
)
```
Another way is to introduce an `Appfile`. You can specify your team there by `team_name "XYZ Corp"`
